clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -758,7 +758,7 @@
                                                                                                  STRING),
                                                                                        new Field("state",
                                                                                                  STRING,
-                                                                                                 "The current state of the group (one of: Dead, Stable, AwaitingSync, or PreparingRebalance, or empty if there is no active group)"),
+                                                                                                 "The current state of the group (one of: Dead, Stable, AwaitingSync, PreparingRebalance, InitialRebalance, or empty if there is no active group)"),
```

Comment:
Do we have to introduce a new state? Could we just piggy-back on `Preparing Rebalance`?